code/modules/mob/living/silicon/robot/robot_modules.dm

```diff
@@ -304,7 +304,8 @@
 		/obj/item/stack/sheet/rglass/cyborg,
 		/obj/item/stack/rods/cyborg,
 		/obj/item/stack/tile/plasteel/cyborg,
-		/obj/item/stack/cable_coil/cyborg)
+		/obj/item/stack/cable_coil/cyborg),
+		/obj/item/holosign_creator/atmos
```

Comment:
code/modules/mob/living/silicon/robot/robot_modules.dm:307:error: ,: expected }
code/modules/mob/living/silicon/robot/robot_modules.dm:282:error: location of top-most unmatched {

fix it by changing line 307 to
/obj/item/stack/cable_coil/cyborg,
and line 308 to 
/obj/item/holosign_creator/atmos)